src/VisualStudio/Core/Def/Implementation/SymbolSearch/RoslynSymbolResult.cs

```diff
@@ -171,81 +163,5 @@ private static Span AsSpan(TextSpan sourceSpan)
         {
             return new Span(sourceSpan.Start, sourceSpan.Length);
         }
-
-        // Taken from AbstractTableDataSourceFindUsagesContext
-        internal static async Task<(Guid projectId, string projectName, SourceText text, Guid documentId)> GetGuidAndProjectNameAndSourceTextAsync(Document document)
-        {
-            // The FAR system needs to know the guid for the project that a def/reference is 
-            // from (to support features like filtering).  Normally that would mean we could
-            // only support this from a VisualStudioWorkspace.  However, we want till work 
-            // in cases like Any-Code (which does not use a VSWorkspace).  So we are tolerant
```

Comment:
why doe ANyCode not use a vsworkspace?  if you don't do that, then features just break.  this sort of approach really worries me as it makes the system feel hacked together with no clear design and with features not working left and right.

presumably users would want filtering.  so we should design a ssytem that works for all these workspaces and all these features.  not just randomly degrade an experience.